types/rebass__forms/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for @rebass/forms 4.0
+// Project: https://github.com/rebassjs/rebass#readme
+// Definitions by: zinozzino <https://github.com/zinozzino>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
```

Comment:
You will have to use TS 3.5 when using the `Omit` 
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type
Is that OK? If so, just update the header and this would remove most of the lint errors

Reply:
i think adding the custom omit type for compatibility is the better way. so I added the one.